src/module_2/module_2.md

```diff
@@ -0,0 +1,3512 @@
+      <th>user_order_seq</th>
+      <th>ordered_items</th>
+      <th>variant_id</th>
+      <th>price</th>
+      <th>compare_at_price</th>
+      <th>vendor</th>
+      <th>product_type</th>
+      <th>tags</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>2215889436804</td>
+      <td>68e872ff888303bff58ec56a3a986f77ddebdbe5c279e7...</td>
+      <td>2020-05-03 21:56:14</td>
+      <td>2020-05-03</td>
+      <td>1</td>
+      <td>33667238658180</td>
+      <td>33667238658180</td>
+      <td>4.19</td>
+      <td>5.1</td>
+      <td>listerine</td>
+      <td>dental</td>
+      <td>[mouthwash]</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>2215889436804</td>
+      <td>68e872ff888303bff58ec56a3a986f77ddebdbe5c279e7...</td>
+      <td>2020-05-03 21:56:14</td>
+      <td>2020-05-03</td>
+      <td>1</td>
+      <td>33667238658180</td>
+      <td>33667238658180</td>
+      <td>4.19</td>
+      <td>5.1</td>
+      <td>listerine</td>
+      <td>dental</td>
+      <td>[mouthwash]</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>2293056798852</td>
+      <td>5842151f9684b07cb153383330d564d6b805488a4e7b4d...</td>
+      <td>2020-06-06 13:03:00</td>
+      <td>2020-06-06</td>
+      <td>1</td>
+      <td>33667238658180</td>
+      <td>33667238658180</td>
+      <td>4.19</td>
+      <td>5.1</td>
+      <td>listerine</td>
+      <td>dental</td>
+      <td>[mouthwash]</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>2293435203716</td>
+      <td>228c4d57135f5610782353dee8709144543ad15216286f...</td>
+      <td>2020-06-06 23:02:28</td>
+      <td>2020-06-06</td>
+      <td>1</td>
+      <td>33667238658180</td>
+      <td>33667238658180</td>
+      <td>4.19</td>
+      <td>5.1</td>
+      <td>listerine</td>
+      <td>dental</td>
+      <td>[mouthwash]</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>2306493612164</td>
+      <td>1296e1e72f7f43ff28d7d285f880ad4d213fa8139233c7...</td>
+      <td>2020-06-21 15:09:26</td>
+      <td>2020-06-21</td>
+      <td>3</td>
+      <td>33667238658180</td>
+      <td>33667238658180</td>
+      <td>4.19</td>
+      <td>5.1</td>
+      <td>listerine</td>
+      <td>dental</td>
+      <td>[mouthwash]</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+print(f"Number of unique items in inventory: {df_inventory['variant_id'].nunique()}")
+print(f"Unique items in inventory: {len(df_inventory)}")
+print(f"Items sold merged with inventory: {len(orders_with_inventory)}")
+
+```
+
+    Number of unique items in inventory: 1733
+    Unique items in inventory: 1733
+    Items sold merged with inventory: 92361
+
+
+We see the value length of orders_with_inventory changes depending which type of merge we perform (left, inner, right, outer). This means that are product being sold that have no corresponde on inventory and products in inventory not being sold. Would be interesting to check those values and verify with Sales team what could be happening there:
```

Comment:
Ah okey cool, so I see that you are aware of this here, great. Still, I think my comment above is probably the most important thing if this review